resources/eventing/values.yaml

```diff
@@ -12,7 +12,7 @@ global:
       version: PR-17124
     certHandler:
       name: api-gateway-webhook-certificates
-      version: "1.4.1"
+      version: "1.5.0"
```

Comment:
We should not bump image in eventing, but rather inform them, that there is a new version available.

Reply:
```suggestion
      version: "1.4.0"
```